modules/ROOT/pages/processes.adoc

```diff
@@ -21,15 +21,15 @@ This feature is only available for the _Administrator_ profile.
 
 The .bar file is ready to be resolved and/or enabled.
 
-You can only install a .bar file that was exported from a Bonita Studio of the same minor (7.x) version as your Bonita Portal and Bonita Engine. +
-If you have a .bar file that was exported from an earlier version of Bonita Studio (7.x-1), it will not work with Bonita Engine. +
+You can only install a .bar file that was exported from a Bonita Studio of the same minor (7.x) version as your Bonita Runtime. +
+If you have a .bar file that was exported from an earlier version of Bonita Studio (7.x-1), it will not work with Bonita Runtime. +
 Export the process definition as a .bos file from your old Bonita Studio, import in into the new Studio, then export it as a .bar. +
 This is necessary for a .bar that has not been deployed. A process that was developed in an older version of Bonita Studio and is already deployed is migrated automatically when you migrate your platform. +
```

Comment:
when you migrate your platform. => when you update your platform.

Reply:
Done